database/migrations/2020_11_01_000300_create_apply_automatically.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Igniter\Coupons\Database\Migrations;
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateApplyAutomatically extends Migration
```

Comment:
Rename class to `AddAutoApplyFieldOnCouponsTable`, change the file name to match too.